recipes/metis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "metis" %}
+{% set version = "5.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: metis-5.1.0.tar.gz
+  md5: 5465e67079419a69e0116de24fce58fe
+  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz
+
+build:
+  skip: true  # [win]
+  number: 0
+  binary_relocation: true
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+
+test:
```

Comment:
This would be a nicer way to do it, but I'm not really picky as long as there are tests.